app/views/search.nunjucks

```diff
@@ -15,6 +15,9 @@
         <!-- label -->
       </label>
 
+      {% if errorMessage %}
```

Comment:
Are you proposing that this PR gets merged and the error message persisting is raised as an issue?